### ISSUE TITLE
Entry: Time

### ISSUE THREAD

**Issue by ryanfox** [NaMoGenMo/2022#3]
I am working on an entry related to movie timelines. I'm using moviepy.

For what I have in mind, detecting scene boundaries is likely the biggest hurdle.

**Comment by ryanfox** [comment]
Progress update: I have transitions stitching together nicely.

Still need to work on detecting scene boundaries.

**Comment by ryanfox** [comment]
I'm a few days late to update this, but I did get something out in time.

I wanted to create a movie with the structure of _Memento_. That is, slice scenes up and interleave them, backwards-forwards-backwards-forwards.

For a substrate, I used Memphis Belle: A Story of a Flying Fortress_, a propaganda film produced by the US government in WWII. Here's the final result:

https://www.youtube.com/watch?v=vkgGLhG3ccI

To create it, I wrote a library called `mementoizer`:
https://pypi.org/project/mementoizer/

Scene detection is pretty rudimentary. Maybe I enhance it in the future.

I wrote a report on the project and result here:
https://foxrow.com/namogenmo-2022

See you in 2023!
